DataServices/Netherlands_Pilot/chloridePredictionService.jsonld

```diff
@@ -0,0 +1,91 @@
+{
+    "id": "urn:ngsi-ld:DataService:ChloridePrediction:3B8E3416-C6ED-40CB-A6F5-8DA08435BB34",
+    "type": "DataService",
+    "title": {
+        "type": "Property",
+        "value": "Chloride prediction service"
+    },
+    "description": {
+        "type": "Property",
+        "value": "Service predicting chloride concentrations in surface water for the next 7 days based on water quality observations"
+    },
+    "algorithm": {
+        "type": "Property",
+        "value": "LSTM"
```

Comment:
It is not an LSTM model, it is a more simpler regression based model. Do we want this information right now? I can ask PWN for it if needed.